### PR TITLE
lwip: port mboxes to core_mbox

### PR DESCRIPTION
**Pull request by miri64**
Ports lwip sys_mbox wrappers to RIOTs native mboxes.

### CODE REVIEW

File: pkg/lwip/include/arch/sys_arch.h [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 #include "cib.h"
 #include "kernel_types.h"
+#include "mbox.h"
 #include "mutex.h"
 #include "random.h"
 #include "sema.h"
```
Comment (kaspar030) [inline pkg/lwip/include/arch/sys_arch.h]: obsolete?

**Comment by kaspar030** [comment]
Can't test, but the code looks good. If it compiles, I trust you tested it. ;) ACK.

**Comment by miri64** [comment]
It does :-). So go? I tested it on native, so there is reason to assume it is working on other platforms, too. ;-)

**Comment by miri64** [comment]
Ping?

**Comment by kaspar030** [comment]
yup!

**Pull request merged**